core/src/main/java/org/geysermc/geyser/GeyserImpl.java

```diff
@@ -121,7 +121,10 @@ public class GeyserImpl implements GeyserApi {
     private FloodgateSkinUploader skinUploader;
     private NewsHandler newsHandler;
 
-    private volatile boolean shuttingDown = false;
```

Comment:
Why remove `volatile` here?

Reply:
I think this was an accidental diff from my debugging.